### PR TITLE
Add emulation of separate chats for whatsapp contact through empty groups

### PR DESCRIPTION
**Pull request by georgeto**
A whatsapp contact can be bound to a group containing the bot. Now each message send in this group will be relayed to the whatsapp contact. And each message sent by the conact will be posted into the group.

### CODE REVIEW

File: wat_bridge/tg.py [hunk 1]
```diff
@@ -110,6 +113,88 @@ def add_contact(message):
 
     tgbot.reply_to(message, 'Contact added')
 
+@tgbot.message_handler(commands=['bind'])
+def add_contact(message):
```
Comment (rmed) [inline wat_bridge/tg.py]: Please change the function name.

File: wat_bridge/tg.py [hunk 2]
```diff
@@ -110,6 +113,88 @@ def add_contact(message):
...
+        tgbot.reply_to(message, 'This group is already bound to ' + current)
+        return
+
+    # Add to database
+    db_set_group(name, group_id)
+
+    tgbot.reply_to(message, 'Bound to group')
+
+@tgbot.message_handler(commands=['unbind'])
+def add_contact(message):
```
Comment (rmed) [inline wat_bridge/tg.py]: Please change the function name.

**Review by rmed — CHANGES_REQUESTED** [review]
Please change the `add_contact` function name for the `/bind` and `/unbind` commands for clarity.

**Comment by rmed** [comment]
Hi, thank you for implementing this! The code looks good and is a very useful feature, although I cannot test the Whatsapp part due to my number being blocked.

Before merging the branch, could you look into the small changes I pointed in the review? Thanks!

**Comment by georgeto** [comment]
Will do, I totally forgot changing the function names.

Have you been blocked for using a instance of wat-bridge?!

**Comment by rmed** [comment]
More like, for using yowsup, apparently.

**Review by rmed — APPROVED** [review]

**Pull request merged**